Features/Context/SubContext/CommonActions.php

```diff
@@ -286,6 +286,18 @@ public function selectFromUniversalDiscovery($path)
     }
 
     /**
+     * @When I select the :path folder in the new Universal Discovery Widget
```

Comment:
"the new XXXX" is a bad idea, at some point there will be a new new XXXX ;-)
joke aside, "in the Finder method of Universal Discovery Widget" would a good way to describe that operation.